Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -173,10 +173,17 @@ void MenuBar::AddViewMenu()
 void MenuBar::AddHelpMenu()
 {
   QMenu* help_menu = addMenu(tr("Help"));
+  QAction* website = help_menu->addAction(tr("Website"));
+  connect(website, &QAction::triggered, this,
+          [=]() { QDesktopServices::openUrl(QUrl(QStringLiteral("https://dolphin-emu.org/"))); });
```

Comment:
Oh, that's because default arguments aren't supported when using a lambda, but we don't need it, so the = is just basically there to ignore the signal argument.

However, the triggered signal HAS no arguments, so the = can be removed. It probably was being copied and pasted before hand, then copy and pasted again here by me.